src/components/questions/DataStore.js

```diff
@@ -1,10 +1,12 @@
 import DataStore from '../datastore/index'
+import mockData from './mockData'
 
 class QuestionsDataStore extends DataStore.Crud {
 
   name = `Question`
   namePlural = `Questions`
   uri = `questions`
+  mockData = mockData
```

Comment:
Storing mock data on each of the `DataStore` now. We use this data in our `mockData` helper later.